sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_diagnostics/diagnostic_logging.py

```diff
@@ -76,16 +76,16 @@ def _initialize(cls):
                     AzureDiagnosticLogging._initialized = True
 
     @classmethod
-    def info(cls, message: str, message_id: int):
+    def info(cls, message: str, message_id: str):
         AzureDiagnosticLogging._initialize()
         _logger.info(message, extra={'msgId': message_id})
 
     @classmethod
-    def warning(cls, message: str, message_id: int):
+    def warning(cls, message: str, message_id: str):
         AzureDiagnosticLogging._initialize()
         _logger.warning(message, extra={'msgId': message_id})
 
     @classmethod
-    def error(cls, message: str, message_id: int):
+    def error(cls, message: str, message_id: str):
```

Comment:
@jeremydvoss 

Changed these to str to suggest string values for diagnostics. This means we should have our messageIds in constants represented as strings. Should be fine right?